app/schemas/species.py

```diff
@@ -17,6 +17,11 @@ class SpeciesBase(BaseModel):
     data_source_id: int
 
 
+class SpeciesBaseIDOnly(BaseModel):
+    id: str
+    data_source_id: int
```

Comment:
Let's remove `data_source_id` too and just fetch species id. It can be obtained from `/api/v1/species/{species_id}` endpoint.

Reply:
I have removed that field now.